sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AbstractTokenContextListener.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.appcenter.utils.context;
+
+/**
+ * Empty implementation to make callbacks optional.
+ */
+public class AbstractTokenContextListener implements  AuthTokenContext.Listener {
```

Comment:
Rename the class to `DefaultTokenContextListner` This isn't an abstract class.

Reply:
or make it abstract to match other classes